src/react-components/auth/AuthContext.js

```diff
@@ -33,6 +33,10 @@ async function checkIsAdmin(socket, store) {
       })
       .receive("error", error => {
         console.error("Error joining Phoenix Channel", error);
+        if (error.reason === "invalid_token") {
+          console.error("Token is invalid. Signing out.");
+          store.update({ credentials: { token: null, email: null } });
```

Comment:
You are doing this in a few spots. Probably good to add `signout()` to store. Though the actual sign out butotn calls `signOut()` on AuthChannel, which we might want to do instead, since it sets a few other things.